doc/install/install-openshift.md

```diff
@@ -0,0 +1,240 @@
+# Install Kuadrant on an OpenShift Cluster
+
+Note these steps need to be taken on each cluster you want to use Kuadrant on.
+
+## Pre Requisites
+
+- OpenShift 4.14.x with community operator catalog available
+- AWS account with route 53 and zone 
+- Accessible Redis Instance
+
+## Setup Environment
+
+```
+export AWS_ACCESS_KEY_ID=xxxxxxx # the key id from AWS
+export AWS_SECRET_ACCESS_KEY=xxxxxxx # the access key from AWS
+export REDIS_URL=redis://user:xxxxxx@some-redis.com:10340
+
+```
+
+## Installing the dependencies
+
+Kuadrant integrates with Istio as a Gateway API provider. Before trying Kuadrant, we need to setup an Istio based Gateway API provider. For this we will use the `Sail` operator. 
+
+- Install v1 of Gateway API
+
+```
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
+```
+
+- ### Install and Configure Istio via the Sail Operator
+
+
+#### Install
+```
+kubectl create ns istio-system
+```
+
+```
+kubectl  apply -f - <<EOF
+kind: OperatorGroup
+apiVersion: operators.coreos.com/v1
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "0"
+  name: sail
+  namespace: istio-system
+spec: 
+  upgradeStrategy: Default  
+---  
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sailoperator
+  namespace: istio-system
+spec:
+  channel: 3.0-dp1
+  installPlanApproval: Automatic
+  name: sailoperator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+EOF
+```
+
+To check the status of the install you can run:
+
+```
+kubectl get installplan -n istio-system -o=jsonpath='{.items[0].status.phase}'
+```
+
+once ready it will be marked `complete` while installing it will be marked `installing`
+
+#### Configure Istio
+
+```
+kubectl apply -f - <<EOF
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+  name: default
+spec:
+  version: v1.21.0
+  namespace: istio-system
+  # Disable autoscaling to reduce dev resources
+  values:
+    pilot:
+      autoscaleEnabled: false
+EOF
+```
+
+Wait for Istio to be ready
+
+```
+kubectl wait istio/default -n istio-system --for="condition=Ready=true"
+```
+
+
+### (Recommended) Thanos and Observability Stack
+
+Kuadrant provides a set of sample dashboards that use the known metrics exported by kuadrant and gateway components to provide insight into the different areas of your APIs and Gateways. While this is not essential it is recommended that you set up an observability stack. Below are links to the OpenShift docs on this and also a link to help with the setup of Thanos for metrics storage.
+
+OpenShift supports a user facing monitoring stack. This can be cofigured and setup via their documentation
+https://docs.openshift.com/dedicated/observability/monitoring/configuring-the-monitoring-stack.html 
```

Comment:
The documentation link points to the openshift dedicated. This expects that openshift has some resources created by default. E.g. `The user-workload-monitoring-config ConfigMap object exists. This object is created by default when the cluster is created.`